addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -499,6 +499,299 @@ registry.slider = publicWidget.Widget.extend({
     },
 });
 
+registry.backgroundVideo = Animation.extend({
+    selector: ".o_video_background",
+    jsLibs: [
+        '/website/static/lib/YTPlayer/jquery.mb.YTPlayer.js',
+    ],
+    disabledInEditableMode: false,
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    start: function () {
+        this._startVideo();
+        return this._super.apply(this, arguments);
+    },
+
+    /**
+     * @override
+     */
+    destroy: function () {
+        this._super.apply(this, arguments);
+        this._stopVideo();
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * When user provides video this method will update the video type according 
+     * to the input.
+     *
+     * @private
+     */
+    _updateVideoType: function () {
```

Comment:
This method seems to be a duplicate of what is done in the editor to parse the video URL... can't we share something in utility functions ?

Reply:
Did you answer that question somewhere? I don't remember